frontend/src/components/Map.jsx

```diff
@@ -0,0 +1,91 @@
+import {useMemo, useCallback, useState, useRef, useEffect} from 'react'
+import {GoogleMap} from '@react-google-maps/api'
+import LocationPin from '@/components/LocationPin'
+import SearchBar from '@/components/SearchBar'
+
+
+// post method
+//TODO: check userID, generate tripID
+
+const postTrip = async (trip) => {
+    console.log(JSON.stringify({"trip": trip}))
+    const response = await fetch('http://localhost:8080/trips/1', {
```

Comment:
This won't work, put the base endpoint url into a variable. Or just always use the Google app engine one